be/src/storage/tablet_updates.cpp

```diff
@@ -2354,7 +2354,8 @@ int64_t TabletUpdates::get_compaction_score() {
         // only 1 input and no delete, no need to do compaction
         return -1;
     }
-    return total_score;
+    // scale score to a reasonable range relative to the number of files * 10
+    return total_score / (config::update_compaction_size_threshold / 10);
```

Comment:
If `update_compaction_size_threshold` less than 10, that will be an execption?

Reply:
done